web/beacon-app/src/application/routes/paths.ts

```diff
@@ -21,6 +21,10 @@ export const ROUTES = {
   FORGOT_PASSWORD: '/forgot-password',
   RESET_PASSWORD: '/reset-password',
   RESET_VERIFICATION: '/reset-verification',
+  DATA_PLAYGROUND: 'https://rotational.io/data-playground/',
+  SDK_DOCUMENTATION: 'https://ensign.rotational.dev/sdk',
+  ENSIGN_UNIVERSITY: 'https://rotational.io/blog/',
```

Comment:
Should the links to `ENSIGN_UNIVERSITY` and `USE_CASES` be moved to `EXTERNAL_LINKS` below to maintain consistency?